CodingDojo/games/expansion/src/main/java/com/codenjoy/dojo/expansion/services/Scores.java

```diff
@@ -92,8 +101,29 @@ public void update(Object data) {
     @Override
     public JSONObject getScore() {
         scores.put(SCORE, score);
+        scores.put(DETAILS, details());
         return scores;
     }
+
+    private String details() {
+        List<Object> list = rounds().toList();
+        int count = list.size();
+        int sum = list.stream()
+                .map(it -> (Integer) it)
+                .mapToInt(it -> it).sum();
+        double average = (count == 0) ? 0 : (100 * sum) / (count * 4.0);
```

Comment:
Have you considered computing an average value by using stream API?
`list.stream().mapToInt(it -> (Integer) it).average()`

Reply:
double average = list.stream()
                .mapToInt(it -> it)
                .average().orElse(0.0) * 25.0;

Reply:
Done 
![image](https://user-images.githubusercontent.com/4629949/135316924-b3cefd83-5b0e-421e-a829-40018ac32fb2.png)